website/docs/sdk-reference/csharp.md

```diff
@@ -45,6 +45,30 @@ else
 }
 ```
 
+:::caution
```

Comment:
I'd move this above the "Sample applications" section to a new section named "Troubleshooting".
It would be awesome if the error message could be shown here too so our customers can identify this problem better.

Reply:
Is [this](https://github.com/configcat/docs/pull/155/commits/cbc18572730d63d8c3aad9ad54ee41ddc22efe6d) what you have in mind?

Reply:
yes :)